k2/csrc/host/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 
 #---------------------------- Build K2 host sources ----------------------------
 
+include_directories(${CUDAToolkit_INCLUDE_DIRS})
```

Comment:
why do we need this for pure host code

Reply:
Oh, I know, as we have used `log.cuh` in host code now.

Reply:
I have removed it. Hope I can make it compiled w/o this line. To be reported in new PR.

Reply:
I have removed it. Hope I can make it compiled w/o this line. To be reported in new PR.